### ISSUE TITLE
Improve the messenger app

### ISSUE THREAD

**Issue by sebastian-code** [vitorfs/bootcamp#105]
Currently the messenger app lies deep inside Bootcamp, and that's perhaps the most attractive feature for the actual user.

It can receive a new workflow to make it more likely other well known apps without loosing the natural attractiveness of Bootcamp.

This workflow improvement its also a chance to address issues #47 and #66 adding a better asynchronous implementation, in this case, the natural way to do it is to implement django-channels with Redis.

**Comment by sebastian-code** [comment]
As I expressed in 8d45249, for now I applied a small tweak to the actual look, I still want to revamp the workflow, but perhaps it will make the application incompatible with previous deployments, affecting actual deployments in production.

For me the main goal of this issue is to make the app similar to new chat apps like Slack or Glip, making things like the message form box to maintain its position at the bottom of the page, rendering the conversation history scrollable. The other point would be to remove the actual user list, which show only those users I had conversations with, and replace it with a real users list, showing the conversation story for each user.
